charts/tezos-reward-distributor/scripts/run.sh

```diff
@@ -20,5 +20,43 @@ python src/main.py \
   --initial_cycle ${INITIAL_CYCLE} \
   -N ${NETWORK} \
   ${EXTRA_TRD_ARGS} \
-  ${dry_run_arg} \
-  ${aep_arg}
+  ${dry_run_arg}
+
+# if TRD fails, send a slack alert
+if [ $? -ne 0 ]; then
+  # check if bot token and channel are set
+  if [ -z "${SLACK_BOT_TOKEN}" ] || [ -z "${SLACK_CHANNEL}" ]; then
+    echo "TRD failed, but SLACK_BOT_TOKEN or SLACK_CHANNEL is not set, failing job"
+    exit 1
+  fi
+  python -c "
```

Comment:
I am a little bit hesitant to give a +1 for this....

Reply:
I was doing it with curl and jq initially, but they are not availbale in the trd alpine container...